exoctk/exoctk_app/app_exoctk.py

```diff
@@ -69,6 +71,7 @@ def index():
 @app_exoctk.route('/limb_darkening', methods=['GET', 'POST'])
 def limb_darkening():
     """The limb darkening form page. """
+    print('safkjhsdkjfhadkjh')
```

Comment:
Was this left over from a test? Can we remove it?

Reply:
Good catch! I'll remove it